app/graphql/mutations/create_cohort.rb

```diff
@@ -23,22 +23,34 @@ class CreateCohort < ApplicationQuery
              }
     argument :ends_at, GraphQL::Types::ISO8601DateTime, required: false
 
-    description 'Create a new cohort'
+    description "Create a new cohort"
 
     field :cohort, Types::CohortType, null: true
 
     def resolve(_params)
-      notify(
-        :success,
-        I18n.t('shared.notifications.done_exclamation'),
-        I18n.t('mutations.create_cohort.success_notification')
-      )
+      cohort = create_cohort
+
+      if cohort.persisted?
+        notify(
+          :success,
+          I18n.t("shared.notifications.done_exclamation"),
+          I18n.t("mutations.create_cohort.success_notification")
+        )
+
+        { cohort: }
```

Comment:
```suggestion
        { cohort: cohort}
```
Woudn't this be more redable for someone new?

Reply:
@bodhish Nah, this is part of the new Ruby 3.1 syntax, that we should get used to. _Punning_ in Rescript, _Shorthand properties_ in JS, and I think they call it _hash value omission_ here. 🤷🏼‍♂️ Same thing, with a slightly more verbose syntax.